### PR TITLE
Use Omaha proxy to find the correct ChromeDriver

### LINKED ISSUE #16448: TaskCluster Chrome builds failing
We last got a master run for Chrome in 4131304.

Ever since, we've got:

```
SessionNotCreatedException: session not created (500): session not created: This version of ChromeDriver only supports Chrome version 76
```

We seem to be running ChromeDriver 76.0.3775.0; but the google-chrome-unstable package is only 75.0.3766.2-1. Canary is currently 76, but given we're running dev on Linux (where there is no canary package), and dev is 75, we end up with this error. Presumably we're downloading ChromeDriver for canary and not dev, and this breaks in the brief periods where canary and dev have different versions?

### PR DESCRIPTION
**Pull request by Hexcles**
Fixes #16448

### CODE REVIEW

File: tools/wpt/browser.py [hunk 1]
```diff
@@ -464,29 +464,32 @@ def find_webdriver(self, channel=None):
...
-                latest_url = "https://chromedriver.storage.googleapis.com/LATEST_RELEASE_%s.%s.%s" % (
-                    parts[0], parts[1], parts[2])
+        assert chrome_version, "Cannot detect the version of Chrome"
+
+        # Remove channel suffixes (e.g. " dev").
+        chrome_version = chrome_version.split(' ')[0]
+        parts = chrome_version.split(".")
+        if len(parts) == 4:
+            latest_url = "https://chromedriver.storage.googleapis.com/LATEST_RELEASE_%s.%s.%s" % (
+                parts[0], parts[1], parts[2])
```
Comment (gsnedders) [inline tools/wpt/browser.py]: Let's just make this `% parts[0:3]`

**Comment by Hexcles** [comment]
@gsnedders I'm going with the second approach I mentioned in the issue: using Omaha proxy to find the revision number so that we can download the correct build of ChromeDriver for that revision from the archive.

**Review by gsnedders — APPROVED** [review]
otherwise LGTM

File: tools/wpt/browser.py (hunk 1 above)
Comment (Hexcles) [reply tools/wpt/browser.py]: Actually, `%` doesn't like lists. I have to convert it to a tuple.

**Pull request merged**